pkg/expect/expect_test.go

```diff
@@ -40,3 +38,17 @@ func TestEcho(t *testing.T) {
 		t.Fatalf("expected error on closed expect process")
 	}
 }
+
+func TestSendLine(t *testing.T) {
+	ep, err := NewExpect("/usr/bin/tr", "a", "b")
```

Comment:
@heyitsanthony I am running `tr (GNU coreutils) 8.21`, but I think Go `exec` library is smart about args so that it is passed as `tr a b`, not `tr "a b"`.